nixos/doc/manual/man-nixos-rebuild.xml

```diff
@@ -425,6 +449,21 @@
     </listitem>
    </varlistentry>
 
+   <varlistentry>
+    <term>
+     <option>--generation</option> <replaceable>generation</replaceable>
+    </term>
+    <listitem>
+     <para>
+      Instead of building a new configuration as specified by
+      <filename>/etc/nixos/configuration.nix</filename>, roll back to a
```

Comment:
nit: `switch` is probably more appropriate terminology here (instead of `roll back`), since the generation could very well be the currently active one, if it was added with `nixos-rebuild boot`.

Reply:
I am all ears for the nitty-gritty details ;)